fe/src/component/bottom/Slides.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import Slide from "./Slide";
+import More from "./More";
+
+const Slides = () => {
+  const [cardsInfor, setCardInfor] = useState([]);
+  const [isDisplayed, setIsDisplayed] = useState(false);
+  const myfetch = (url) => {
+    return fetch(url).then((res) => res.json());
+  };
+
+  useEffect(() => {
+    const url = [
+      "https://api.codesquad.kr/onban/side",
+      "https://api.codesquad.kr/onban/soup",
+      "https://api.codesquad.kr/onban/main",
+    ];
+
+    const data = Promise.all(
+      url.map(async (v) => {
+        return await myfetch(v);
+      })
+    );
+
+    data.then((res) => setCardInfor(res));
+  }, []);
+
+  const onChangeSlideDisplay = () => {
+    isDisplayed ? setIsDisplayed(false) : setIsDisplayed(true);
```

Comment:
```suggestion
    setIsDisplayed(!isDisplayed);
```
boolean 값이면 바로 넣어줄수 있겠네용